test/acceptance/features/serviceClaim.feature

```diff
@@ -747,3 +747,83 @@ Feature: Service claim with label selector
         And  On Worker Cluster "worker", Service Binding "sc-test" does not exist in "applications"
         And  On Worker Cluster "worker", the secret "sc-test" does not exist in namespace "applications"
 
+    Scenario: Create ServiceClaim with ApplicationClusterContext and EnvironmentTag
```

Comment:
I'd propose to implement this test in `serviceclaim_webhook_test.go`. WDYT?

Reply:
@filariow 
There is a test : https://github.com/primaza/primaza/pull/83/files#diff-e412822a13ddd16751410c7c39a421605cca061013c078baf4fa953e7e8cdfc0R40
Please check!

Reply:
right, my concern here is that this test does not have the 3 steps Given-When-Then and that's validating the ServiceClaim's yaml. We may have this kind of validation just in `serviceclaim_webhook_test.go` without duplicating it. Or may it be a good reason to check it also here?

Reply:
The test in the `serviceclaim_webhook_test.go` is a unit test. So it has the minimum required test fixture for the test. What we are writing using these feature files is functional/acceptance tests. These are complementary.

Reply:
ok